__tests__/app.test.js

```diff
@@ -1,41 +1,80 @@
-    test("200: Responds with an object detailing the documentation for each endpoint", () => {
-        return request(app)
-            .get("/api")
-            .expect(200)
-            .then(({body: {endpoints}}) => {
-                expect(endpoints).toEqual(endpointsJson);
-            });
-    });
+  test("200: Responds with an object detailing the documentation for each endpoint", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body: { endpoints } }) => {
+        expect(endpoints).toEqual(endpointsJson);
+      });
+  });
 });
 
-describe('GET /api/topics', () => {
-    it('should respond with status 200 and an array of topic objects', () => {
-        return request(app).get("/api/topics").expect(200).then(({body: {topics}}) => {
-            topics.forEach((topic) => {
-                expect(topic).toHaveProperty("slug")
-                expect(topic).toHaveProperty("description");
-                expect(topic).toMatchObject({
-                    description: expect.any(String),
-                    slug: expect.any(String),
-                })
-            })
-        })
-    });
-});
\ No newline at end of file
+describe("GET /api/topics", () => {
+  it("should respond with status 200 and an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("should return status 200 and an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  it("should respond with 404 not found if the article does't exist", () => {
+    return request(app)
+      .get("/api/articles/0")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not found");
+      });
+  });
+  it('should respond with status 400 and message "Bad request" if the id is NaN', () => {
+    return request(app)
+      .get("/api/articles/one")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
```

Comment:
Nice, but the description here can be more clear. This is not specifically `NaN`, but just any value that isnt a 'number'. The wording seems like you are specifically looking for the value `NaN`

Reply:
NaN means not a number, just used the acronym